src/main/resources/com/puppycrawl/tools/checkstyle/grammars/javadoc/JavadocParser.g4

```diff
@@ -877,20 +877,20 @@ reference:
       ;
 parameters: LEFT_BRACE (ARGUMENT | COMMA | WS | NEWLINE | LEADING_ASTERISK)* RIGHT_BRACE;
 
-javadocTag: AUTHOR_LITERAL (WS | NEWLINE)* ((WS | NEWLINE) description)?
+javadocTag: AUTHOR_LITERAL (WS | NEWLINE)+ description
```

Comment:
If there is a newline, doesn't it require a check for `LEADING_ASTERISK`?
Some of the others follow the same.

Reply:
It will go into `description`.